aws/resource_aws_lb_target_group_test.go

```diff
@@ -118,6 +118,7 @@ func TestAccAWSLBTargetGroup_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_lb_target_group.test", "stickiness.0.cookie_duration", "10000"),
 					resource.TestCheckResourceAttr("aws_lb_target_group.test", "health_check.#", "1"),
 					resource.TestCheckResourceAttr("aws_lb_target_group.test", "health_check.0.path", "/health"),
+					resource.TestCheckResourceAttr("aws_lb_target_group.test", "health_check.0.enabled", "true"),
```

Comment:
It would be good to see some additional test cases around healthcheck enabled = true/false. I believe we have an explicit test case for HealthCheckInterval we should probably have one explicitly for HealthCheck.

Reply:
I just added a couple of tests.